### PR TITLE
Removal of change password menu entry

### LINKED ISSUE #2229: Remove "change password"
Change password is currently changing only the password with which the app authenticates to the server.
If this password is not valid anylonger, e.g. when the auth token is revoked, the user gets the Authenticator/Login screen presented. So this is all fine.

Currently when clicking on "change password" this just opens up AuthenticatorActivity / Weblogin flow, which can change the internal stored password, but as seen above, this is useless.
A real password changing is only possible via webUI and should only be done there, especially if other user backends are handled there.

Therefore I suggest to remove the "change password" option.

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Resolves #2229 by removing the change password option in the user info screen

cc @mario @tobiasKaminsky

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
:+1:

**Comment by AndyScherzinger** [comment]
@mario ping for review ;-)

File: src/main/res/menu/user_info_menu.xml [hunk 1]
```diff
@@ -1,10 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
+<!--
+  Nextcloud Android client application
 
-    <item
-        android:id="@+id/change_password"
-        android:title="@string/change_password">
-    </item>
+  Copyright (C) 2017 Andy Scherzinger
```
Comment (mario) [inline src/main/res/menu/user_info_menu.xml]: 2018 now? xD

**Comment by mario** [comment]
One small comment, then merge.

**Pull request merged**